tests/python/contrib/test_ethosu/test_legalize.py

```diff
@@ -875,6 +875,91 @@ def verify(ext_func):
     verify(mod["tvmgen_default_ethos_u_main_0"])
 
 
+def test_tflite_pool2d_extra_strides_legalize():
```

Comment:
This legalization test is fine, it checks whether the pooling matching the specific conditions gets partitioned for microNPU. I think it would be good to also test that it gets correctly legalized to `ethosu.pooling` op (i.e. the strides will be correctly replaced with [1, 1]).